drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/DMNTypeServiceImpl.java

```diff
@@ -61,7 +62,6 @@ public FactModelTuple retrieveFactModelTuple(Path path, String dmnPath) {
         for (InputDataNode input : dmnModel.getInputs()) {
             DMNType type = input.getType();
             checkTypeSupport(type, errorHolder, input.getName());
-            visitCompositeType(type, false, errorHolder, input.getName());
```

Comment:
@yesamer could you please explain why we remove this `visitCompositeType`? I thought in my review we just discuss `HashSet` vs `TreeSet`

Reply:
@jomarko Because, after a discussion with Gabriele, we discovered that   `checkTypeSupport` method simply calls `visitCompositeType` with the same parameters, and this generate the issue: `visitCompositeType` is called 2 times.